recipes/amqp-cpp/all/test_package/conanfile.py

```diff
@@ -0,0 +1,15 @@
+from conans import ConanFile, CMake
+import os
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        bin_path = os.path.join("bin", "test_package")
```

Comment:
Please exclude running for cross-compiling https://github.com/bincrafters/templates/blob/ebf1bda47919f25aadce062e75855f515694aafd/default/test_package/conanfile.py#L15

Reply:
Done